docs/docs/getting-started/users-and-authentication.md

```diff
@@ -35,17 +48,25 @@ identifier and can be granted a combination of [scopes](#scopes).
 :::note
 
 The email address used to create a new user is only used as an identifier and is unrelated to the email address used for
-[licensing](installation/licensing.md). Users are defined locally to a specific AICA System Configuration, and
-access scopes or passwords are not inferred or shared between different configurations, even if the same email address
-is used.
+[licensing](installation/licensing.md). Users are defined locally to a specific AICA System Configuration, and access
+scopes or passwords are not inferred or shared between different configurations, even if the same email address is used.
 
 :::
 
+If AICA Studio is accessed from a browser external to AICA Launcher, or if the user of AICA Launcher logs out of the
+current session, a valid user email and password must be supplied to log back in to AICA Studio. To create a new user
```

Comment:
I feel that these two sentences don't go together. The first one is breaking the flow of how one should create a user. I think the first sentence could be a tip or a note at the end of this subsection, after the image.

"If AICA Studio is accessed from a web browser, or if the user logs out of the current AICA Studio session, a valid user email and password must be supplied to log in to AICA Studio."